pages/content_script_loader.coffee

```diff
@@ -0,0 +1,28 @@
+injectContentScripts = ->
```

Comment:
No, sorry. The revert commit reintroduced it and I didn't notice. I'll remove it